android/Cargo.toml

```diff
@@ -12,18 +12,19 @@ crate-type = ["cdylib"]
 
 [dependencies]
 log = "0.4"
-itertools = "0.9.0"
-ironoxide = { version = "~0.23.1", features = ["blocking", "beta", "tls-rustls"], default-features = false }
+itertools = "0.10.0"
+ironoxide = { version = "0.26", features = ["blocking", "beta", "tls-rustls"], default-features = false }
 chrono = "0.4"
-serde_json = "~1.0"
+serde_json = "1.0"
 jni-sys = "0.3.0"
+tokio = { version = "1", features = ["rt-multi-thread"] }
```

Comment:
This is needed due to a bug in `ironoxide` before version `0.27` (it calls tokio's `new_multi_threaded` function). That version also gets rid of `chrono`, so it was a larger change that I was trying to not include.